src/releng_docs/shipit_signoffs

```diff
@@ -0,0 +1 @@
+../shipit_signoff/docs
```

Comment:
We talked about this a bit in #230, and doing the symlink the other way means that "git log" and other things that operate on subdirectories of a repository not work as well.

Perhaps app docs should be separate altogether, and releng_docs could link to them?